openfisca_uk/variables/finance/tax/income_tax/liability.py

```diff
@@ -488,7 +488,10 @@ class is_higher_earner(Variable):
 
     def formula(person, period, parameters):
         income = person("adjusted_net_income", period)
-        return person.get_rank(person.benunit, -income) == 0
+        return (
+            person.get_rank(person.benunit, -income + random(person) * 1e-2)
```

Comment:
could you leave a comment describing this?

Reply:
Yep. In case it's not clear here, I was wondering whether if partners who have exactly the same income would mistakenly both be liable for the HITC, which is paid by the higher earner. But this didn't work and I'm pretty sure it's handled somewhere else also.